### ISSUE TITLE
[GUI] {"exception":"No concept with ID [655584] exists in keyspace [words]"}

### ISSUE THREAD

**Issue by pouledodue** [graknlabs/grakn#1514]
`{"exception":"No concept with ID [655584] exists in keyspace [words]"}`
when I click on a relation in the GUI

**Comment by marco-scoppetta** [comment]
Hi @pouledodue,

can you please provide the query you are executing to get this faulty relations?

So that we can try to reproduce and investigate,

thanks

**Comment by pouledodue** [comment]
hi, just use this ontology:

```
insert

##################################################################
# Schema
##################################################################

word sub entity
    plays subtype
    plays supertype
    plays prep
    plays pobj
    has name;

f sub word;
o sub word;

pobj sub role;
prep sub role;
subtype sub role;
supertype sub role;

inheritance sub relation
    relates supertype
    relates subtype;

pair sub relation
    relates prep
    relates pobj
    has typ
    has name;

name sub resource datatype string;
typ sub resource datatype string;


##################################################################
# Data
##################################################################

$f isa f, has name "f";
$o isa o, has name "o";

$aa isa word, has name "aa";
$bb isa word, has name "bb";
$cc isa word, has name "cc";

(supertype: $o, subtype: $aa) isa inheritance;
(supertype: $o, subtype: $bb) isa inheritance;
(supertype: $o, subtype: $cc) isa inheritance;

$pp isa word, has name "pp";
$qq isa word, has name "qq";
$rr isa word, has name "rr";
$rr2 isa word, has name "rr";

(supertype: $f, subtype: $pp) isa inheritance;
(supertype: $f, subtype: $qq) isa inheritance;
(supertype: $f, subtype: $rr) isa inheritance;
(supertype: $f, subtype: $rr2) isa inheritance;

##################################################################
# Inference: all pairs;
##################################################################

isa inference-rule
    lhs {
        $x isa word;
        $y isa word;
        $x has name != 'f';
        $x has name != 'o';
    }
    rhs {
        (prep: $x, pobj: $y) isa pair;
    };

##################################################################
# Inference: pairs FF
##################################################################

isa inference-rule
    lhs {
        $f isa f;
        (subtype: $prep, supertype: $f) isa inheritance;
        (subtype: $pobj, supertype: $f) isa inheritance;
        $p (prep: $prep, pobj: $pobj) isa pair;
    }
    rhs {
        $p has name 'ff';
    };

##################################################################
# Inference: pairs FO
##################################################################

isa inference-rule
    lhs {
        $f isa f;
        $o isa o;
        (subtype: $prep, supertype: $f) isa inheritance;
        (subtype: $pobj, supertype: $o) isa inheritance;
        $p (prep: $prep, pobj: $pobj) isa pair;
    }
    rhs {
        $p has name 'fo';
    };
```

then go in the dashboard, activate inference, then click on a `pair` relation
after querying this: 
`match $p isa pair, has name 'ff'; offset 0; limit 30;`

**Comment by pouledodue** [comment]
the error will in a red box in the dashboard, or in logs/ or in the javascript console (code 500)

**Comment by pouledodue** [comment]
I use the keyspace 'words' though it probably don't matters

**Comment by marco-scoppetta** [comment]
Hi, this is fixed in [#1529](https://github.com/graknlabs/grakn/pull/1529)

**Comment by pouledodue** [comment]
thanks
